app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -0,0 +1,98 @@
+<ng-form name="persistentVolumeClaimForm">
+  <fieldset ng-disabled="routingDisabled">
+
+    <!-- Name -->
+    <div ng-show="showNameInput" class="form-group">
+      <label for="claim-name" class="required">Name</label>
+      <!-- regex, maxlength from k8s.io/kubernetes/pkg/util/validation/validation.go -->
+      <input
+        id="claim-name"
+        class="form-control"
+        type="text"
+        name="name"
+        ng-model="claim.name"
+        ng-required="showNameInput"
+        ng-pattern="/^[a-z0-9]([-a-z0-9]*[a-z0-9])?$/"
+        ng-maxlength="63"
+        ng-minlength="2"
+        placeholder="my-claim-name"
+        select-on-focus
+        autocorrect="off"
+        autocapitalize="off"
+        spellcheck="false"
+        aria-describedby="claim-name-help">
+      <div>
+        <span id="claim-name-help" class="help-block">A unique name for the storage claim within the project.</span>
+      </div>
+      <div class="has-error" ng-show="persistentVolumeClaimForm.name.$error.pattern && persistentVolumeClaimForm.name.$touched && !routingDisabled">
+        <span class="help-block">
+          Claim names may only contain lower-case letters, numbers, and dashes.
+          They may not start or end with a dash.
+        </span>
+      </div>
+    </div>
+
+    <div class="form-group" >
+      <label class="required">Access Mode</label><br/>
+      <!--modes = { 'RWX':'Read/Write Many', 'RWO':'Read/Write Once', 'ROX':'Read-Only Many' };-->
+      <div class="radio" ng-app ng-init="claim.accessModes=ReadWriteOnce">
```

Comment:
Why is `ng-app` on this element?

I'd rather initialize the value once in the controller rather than using `ng-init`.